test/onnx/onnx_import_provenance.in.cpp

```diff
@@ -85,11 +85,11 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, provenance_multiple_outputs_op)
 NGRAPH_TEST(onnx_${BACKEND_NAME}, provenance_tagging_constants)
 {
     test_provenance_tags<default_opset::Constant>("onnx/provenance_input_tags.prototxt",
-                                                  "<ONNX Input (initializer_of_A) Shape{0}>");
+                                                  "<ONNX Input (initializer_of_A) {1}>");
```

Comment:
Let's add `Shape:` before the shape in this case. Will be easier to read if shape is fully dynamic.